src/components/parameter-pages/organize-output-by-swtich.tsx

```diff
@@ -0,0 +1,20 @@
+'use client';
+
+import { organizeOutputByPatientState } from '@/state/organize-output-by-patient-state';
+import { Switch } from '@mantine/core';
+import { useRecoilState } from 'recoil';
+
+export default function OrganizeOutputByPatientSwitch() {
+  const [organizeOutputByPatient, setOrganizeOutputByPatient] = useRecoilState(organizeOutputByPatientState);
+
+  return (
+    <Switch
+      label="Add organizeOutputBy=Patient to the query string"
```

Comment:
the `<Code>` tag could help with readability for the label 

ex:
`label= {<Text>Add <Code>organizeOutputBy=Patient</Code>to the query string?</Text>}`
<img width="483" alt="Screenshot 2025-01-27 at 11 27 44 AM" src="https://github.com/user-attachments/assets/5b48800d-7b41-4a79-8080-6ffd12ae6702" />



really doesnt matter much but food for thought

Reply:
Good call!!!